rpc-server/src/utils.rs

```diff
@@ -146,20 +145,27 @@ pub async fn get_current_validators(
     Ok(near_rpc_client.call(params, None).await?)
 }
 
+pub async fn get_current_protocol_version(
```

Comment:
btw could you introduce the metric with protocol version to expose, just in case